gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/equality/EqualityTest.java

```diff
@@ -126,10 +132,11 @@ public void testCollectionEqualityByGraphElementData() throws Exception {
   public void testCollectionEqualityByGraphData() throws Exception {
     FlinkAsciiGraphLoader loader = getTestGraphLoader();
 
-    CollectionEquality equality = new CollectionEquality(
-      new GraphHeadToDataString(),
-      new VertexToDataString(),
-      new EdgeToDataString(),
+    CollectionEquality<EPGMGraphHead, EPGMVertex, EPGMEdge, LogicalGraph, GraphCollection> equality =
+      new CollectionEquality<>(
```

Comment:
Indentation: arguments to CollectionEquality constructor should have more indentation.